src/main/java/eu/fasten/maven/CheckMojo.java

```diff
@@ -57,17 +49,11 @@
 import org.json.JSONObject;
 import org.json.JSONTokener;
 
-import eu.fasten.analyzer.javacgopal.data.CallGraphConstructor;
-import eu.fasten.analyzer.javacgopal.data.PartialCallGraph;
-import eu.fasten.analyzer.javacgopal.data.exceptions.OPALException;
-import eu.fasten.core.data.ExtendedRevisionJavaCallGraph;
-import eu.fasten.core.data.JavaScope;
-import eu.fasten.core.merge.LocalMerger;
-import eu.fasten.maven.analyzer.MavenRiskContext;
-import eu.fasten.maven.analyzer.RiskAnalyzer;
-import eu.fasten.maven.analyzer.RiskAnalyzerConfiguration;
-import eu.fasten.maven.analyzer.RiskContext;
-import eu.fasten.maven.analyzer.RiskReport;
+import java.io.*;
```

Comment:
I'm really not a fan of this kind of import (especially when the package contains so many classes), I much prefer explicitly listing the classes you actually use.

Reply:
Yeah, me neither. The IDE sneaked it in and I missed it. I'll fix it :-)